src/Concerns/InteractsWithExpiration.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace RyanChandler\Bearer\Concerns;
+
+use RyanChandler\Bearer\Models\Token;
+
+trait InteractsWithExpiration
+{
+    /**
+     * Add months to the expiration date.
+     */
+    public function addMonths(int $months): Token
+    {
+        $expires = $this->expires_at ?: now();
+
+        return $this->expires($expires->addMonths($months));
+    }
+
+    /**
```

Comment:
We don't need any of these DocBlocks. Everything is typed and the methods are clearly named.